Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tensorflow/tensorflow:2.3.0-gpu
+
+RUN apt-get update \
+&& apt-get upgrade -y \
+&& apt-get install -y \
+&& apt-get -y install apt-utils gcc libpq-dev libsndfile-dev git build-essential cmake screen
+
+# Clear cache
+RUN apt clean && apt-get clean
+
+# Install dependencies
+COPY requirements.txt /
+RUN pip --no-cache-dir install -r /requirements.txt
+
+# Install rnnt_loss
+COPY scripts /scripts
+RUN export CUDA_HOME=/usr/local/cuda \
```

Comment:
@fengredrum We should have an option to install this rnnt_loss, like having an `ENV use_warprnnt` so that we can choose to install warp transducer loss, or use rnnt loss in pure tf by not installing warp-transducer